src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -12,7 +12,7 @@
     <Compile Include="CSharpArgumentInfoTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\src\Microsoft.CSharp.csproj">
+    <ProjectReference Include="..\pkg\Microsoft.CSharp.pkgproj">
```

Comment:
I have 3.5-beta but I am still seeing the errors in System.Collections tests. I'll try to re-install the plugin and try again. 

you are right regarding running the test, when specifying the full path for corerun.exe, it launch the test.